### ISSUE TITLE
General Query window

### ISSUE THREAD

**Issue by starfishmod** [le-yams/nbmongo#61]
Ideally I'd like a place where I can write my own complex queries and statements without the restrictions of the criteria/projection/sort window or complex updates and deletes. 

i.e db.mycollection.find({some criteria}).forEach(function(res){...})

I couldn't find a way to do this. Am I missing something or is there a way to achieve this?

**Comment by le-yams** [comment]
Hi starfishmod,

Unfortunately the plugin doesn't support this feature. But indeed this could be something nice to have.
Maybe something like adding the ability to save js scripts as "MongoDB script" and so they could be run against any mongo database.

Regards,
Yann

**Comment by starfishmod** [comment]
@le-yams yes something that I can edit and run scripts would be perfect. Obviously it would be awesome with syntax highlighting etc :)
Thanks,
Andrew

**Comment by Chris2011** [comment]
This is really a needed feature, I got a query to execute against a mongo DB and I was searching minutes and now I see that this is not implemented... I have similar query: db.getCollection('whatever').update(...
